src/templates/static/legal/complaints.jsx

```diff
@@ -5,7 +5,7 @@ const Complaints = () => (
         <h2 data-anchor='complaints-and-disputes'>{it.L('Complaints and disputes')}</h2>
         
         <p>{it.L('If the client would like to file a complaint with regards to the Company\'s service, the client can contact the Company at [_1], providing any relevant details relating to the client\'s complaint. [_2]If you are registered with Deriv (Europe) Limited, you can also place a complaint by calling [_3].[_4] The Company will investigate each inquiry and provide a final response in the shortest time possible.',
-            '<a href=\'mailto:complaints@binary.com\'>complaints@binary.com</a>', '<span data-show="malta">', '<a href=\'call:+447723580049\'>+447723580049</a>', '</span>')}
+            '<a href=\'mailto:complaints@binary.com\'>complaints@binary.com</a>', '<span data-show="eucountry">', '<a href=\'call:+447723580049\'>+447723580049</a>', '</span>')}
```

Comment:
this makes it visible to MF as well but the description of the task says only MLT and MX.

Reply:
@negar-binary `data-show="malta, iom"` is correct for this?

Reply:
@ali-fs yep, that's right